guides/plugins/apps/storefront/customize-templates.md

```diff
@@ -0,0 +1,99 @@
+---
+nav:
+  title: Customize templates
+  position: 10
+
+---
+
+# Customize Templates
+
+## Overview
+
+This guide will cover customizing Storefront templates using app.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+* A basic understanding of [Shopware app development](../app-base-guide).
+* Familiarity with the [Twig template](https://twig.symfony.com/) is beneficial.
+
+## Getting started
+
+This guide assumes you have already set up your Shopware app. If not, refer to the [app base guide](../app-base-guide) for the initial setup.
+
+The following sections gives you a very short example on how you can extend a storefront block. For simplicity sake, only the page logo is replaced with a 'Hello world!' text.
+
+### Setting up app's view directory
+
+First of all, in your app's root, register your app's own view path, which basically represents a path in which Shopware 6 is looking for template-files. By default, Shopware 6 is looking for a directory called `views` in your app's `Resources` directory, so the path could look like this: `<app root>/src/Resources/views`
+
+### Locating the template
+
+As mentioned earlier, this guide is only trying to replace the 'demo' logo with a 'Hello world!' text. In order to find the proper template, you can simply search for the term 'logo' inside of the `<shopware root>/src/Storefront` directory. This will eventually lead you to [this file](https://github.com/shopware/shopware/blob/v6.3.4.1/src/Storefront/Resources/views/storefront/layout/header/logo.html.twig).
```

Comment:
```suggestion
As mentioned earlier, this guide is only trying to replace the 'demo' logo with a 'Hello world!' text. In order to find the proper template, you can simply search for the term 'logo' inside the `<shopware root>/src/Storefront` directory. This will eventually lead you to [this file](https://github.com/shopware/shopware/blob/v6.3.4.1/src/Storefront/Resources/views/storefront/layout/header/logo.html.twig).
```